usage/results/plugin_report.rst

```diff
@@ -1,13 +1,13 @@
 Understanding plugin report
 ===========================
 
-For better organization, all the plugins of same test code are grouped together. When you open a
+For better organization, all the plugins of the same test code are grouped together. When you open a
```

Comment:
Remove the "the" before "plugins".
"For better organization, all plugins of the same test code are grouped together" sounds much better.